### PR TITLE
Switch to Symfony's version of the Path helper

### PR DESCRIPTION
**Pull request by m-vo**
The `webmozart/path-util` package got abandoned but the `Path` helper class was added to the `Filesystem` component of Symfony 5.4. 

Besides the `getFilename()` method removed (which is basically `basename`) and the use of a variadic instead of an array parameter in `getLongestCommonBasePath()` everything stayed the same.

The CI jobs must be rerun once Symfony 5.4 is released.

### CODE REVIEW

**Review by ausi — DISMISSED** [review]

**Review by Toflar — DISMISSED** [review]

**Review by bytehead — DISMISSED** [review]

File: core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php [hunk 1]
```diff
@@ -664,7 +664,7 @@ private function mockFilemtime(array $pathToMtime): void
                     namespace %s;
 
                     function filemtime(string $filename) {
-                        if (null !== ($mtime = unserialize('%s')[\Webmozart\PathUtil\Path::canonicalize($filename)] ?? null)) {
+                        if (null !== ($mtime = unserialize('%s')[\Symfony\Component\Filesystem\Path::canonicalize($filename)] ?? null)) {
```
Comment (leofeyer) [inline core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php:667]: Why are we using a FQCN here?

Comment (m-vo) [reply core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php:667]: It's part of a string. We could also write an import, but this looks simpler to me.

Comment (leofeyer) [reply core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php:667]: > It's part of a string.

Really? It seems like an array key to me. 🤔

Comment (ausi) [reply core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php:667]: @leofeyer the whole code block is part of a string (using the nowdoc syntax), see line 663: `<<<'EOPHP'`

**Comment by aschempp** [comment]
@m-vo 4.x now requires Symfony 5.4, can you please resolve the merge conflicts?

**Review by leofeyer — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Pull request merged**